### PR TITLE
[53767] Added nuget package and updated version

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

**Review by pavshr — APPROVED** [review]

**Review by pavshr — APPROVED** [review]
Even better :)

**Review by pavshr — CHANGES_REQUESTED** [review]
Set also ENApiVersion to 2 in Info.plist

File: NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj [hunk 1]
```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Configurations>Debug;Release;UNIT_TEST</Configurations>
+    <RestoureSources>
+        $(MSBuildThisFileDirectory);
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj:7]: Seem to be not resolved automatically without explicitly adding nuget source in VS.

File: NDB.Covid19/NDB.Covid19.iOS/Info.plist [hunk 2]
```diff
@@ -108,6 +108,6 @@
 	<key>ENDeveloperRegion</key>
 	<string>NO</string>
 	<key>ENAPIVersion</key>
-	<string>1</string>
+	<string>2</string>
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19.iOS/Info.plist:111]: change to integer, string in new nuget gives error

**Review by pavshr — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.iOS/Info.plist (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.iOS/Info.plist]: Done

File: NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj]: Been through various iterations, and it now seems to be working both for me locally and in the pipeline

**Review by pavshr — APPROVED** [review]
Approved :)

**Pull request merged**